projects/ng-rule-editor/src/lib/rule-editor.service.spec.ts

```diff
@@ -2,7 +2,11 @@ import { TestBed } from '@angular/core/testing';
 
 import { RuleEditorService } from './rule-editor.service';
 
-import * as mockdata from '../../../../src/app/mock-data';
+import {phq9ScoreLinkId, fhir, phq9Vars, varTemp, phq9Val, phq9Score} from '../../../../src/app/mock-data';
+
+function insert(values) {
```

Comment:
I think the creation of the mock-data variables would be better off in the mock-data file, so they could be used in other test files if needed.  You would export a single array containing the variables, rather than var1, var2, etc.

Reply:
Fixed